atlasdb-client/src/main/java/com/palantir/atlasdb/keyvalue/impl/ProfilingKeyValueService.java

```diff
@@ -89,32 +91,38 @@ public static ProfilingKeyValueService create(KeyValueService delegate, long slo
             int numCells, long sizeInBytes) {
         return (logger, stopwatch) ->
                 logger.log("Call to KVS.{} on table {} for {} cells of overall size {} bytes took {} ms.",
-                        method, tableRef, numCells, sizeInBytes, stopwatch.elapsed(TimeUnit.MILLISECONDS));
+                        SafeArg.of("method", method), UnsafeArg.of("tableRef", tableRef),
```

Comment:
We can use `LoggingArgs.tableRef` here, which uses safe/unsafe depending on whether you've specified the table as safe